src/index.html

```diff
@@ -15,7 +15,7 @@ <h1>Today's To Do</h1>
       </div>
       <ul id="todo-list"></ul>
       <div class="button-container">
-        <button id="completed-button" class="complete">Clear as Completed</button>
+        <button id="completed-button" class="complete">Clear as Complete</button>
       </div>
```

Comment:
It seems that you added all functionality for the TODO list app, Kindly don't get ahead of the project milestone progress.
Unfortunately, you missed some requirements for this milestone:

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Webpack%20Plugin.-,Create%20an%20index.js%20file%20and%20set%20an%20array%20of%20some%20simple,-to%20do%20tasks), Please create an array of simple todos tasks and render them on the DOM. Each task object should contain three keys:
   - description [string].
   - completed [bool].
   - index: [number].

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=CSS%20Loader.-,Your%20list%20should%20be%20a%20clone%20of%20the%20part%20of%20the%20minimalist%20project%20captured%20in%20the%20video%20below.,-All%20your%20source), Please make sure that your app matches the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/SAZZAD1Q2/To-Do-List/assets/68796232/eb2b8839-11e1-42d7-939b-79dd5b510e37)  | ![image](https://github.com/SAZZAD1Q2/To-Do-List/assets/68796232/4625b9a3-df77-4a4f-b008-3deec1ea69cc)  |